dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/CacheConfigManager.java

```diff
@@ -23,95 +18,51 @@ public class CacheConfigManager {
 
     private static final CacheConfigManager CACHE_CONFIG_MANAGER = new CacheConfigManager();
 
-    private static LoadingCache<String, String> installationAccessTokenCache;
+    private static LoadingCache<Long, GitHub> githubClientAPICache;
```

Comment:
I **think** this should technically be `volatile`, for the same reasons as mentioned in my other comment, even though I was wrong about those particular fields. It's written to be the init thread, but read from request threads.

In practice, it's going to be written to only once, so I don't doubt any request threads wouldn't have thT value, but in theory it's possible.

Reply:
Well, the `LoadingCache` presumably is instantiated only once and a reference given to `githubClientAPICache`. Presumably the `GitHub` clients inside are also only written once and loading cache claims to be [thread-safe ] (https://guava.dev/releases/21.0/api/docs/com/google/common/cache/LoadingCache.html)

On the other hand, I can't see how adding `volatile` could possibly hurt, so adding :shrug: